packages/suite/src/views/onboarding/steps/SelectBackupType.tsx

```diff
@@ -182,17 +200,57 @@ type FloatingSelectionsProps = {
     defaultType: BackupType;
 };
 
-const DefaultTag = () => (
-    <Badge variant="primary" inline margin={{ left: spacings.xxs }}>
-        <Text typographyStyle="hint">
-            <Translation id="TR_ONBOARDING_BACKUP_TYPE_DEFAULT" />
-        </Text>
-    </Badge>
-);
+const DefaultTag = () => {
+    const { isMobileLayout } = useLayoutSize();
+
+    return (
+        <Badge
+            variant="primary"
+            inline
+            margin={{ left: spacings.xxs }}
+            size={isMobileLayout ? 'tiny' : undefined}
+        >
+            <Text typographyStyle="hint">
+                <Translation id="TR_ONBOARDING_BACKUP_TYPE_DEFAULT" />
+            </Text>
+        </Badge>
+    );
+};
+
+const UpgradableToMultiTag = () => {
+    const { isMobileLayout } = useLayoutSize();
+
+    return (
+        <Badge
+            variant="tertiary"
+            inline
+            margin={{ left: spacings.xxs }}
```

Comment:
btw I think it's fine use just `left: 4`, it doesn't allow other numbers except spacing values.